deploy/charts/templates/service.yaml

```diff
@@ -12,7 +12,6 @@ spec:
   type: {{ .Values.service.type }}
   ports:
   - port: {{ .Values.service.port }}
-    targetPort: {{ .Values.service.targetPort }}
```

Comment:
I agree with removing the `service.targetPort` value, but I think we should still specify `targetPort` in service specs as otherwise we're relying on the Kubernetes default (which defaults to be equal to `port`. I prefer to use port aliases for this. So, because we named the port to be `service.portName` in the deployment spec, I would recommend specifying the `targetPort` to also be `service.portName` in the service spec.

Reply:
sure. can do